api/src/migrations/2-subscribers.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('subscribers', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      email: {
+        type: Sequelize.TEXT
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE
+      }
+    });
+  },
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.dropTable('subscribers');
+  }
+}
```

Comment:
These will be for the users who want to get notified of when there is someone in the area getting signatures

Reply:
whats the reason behind the file being called 2-subscribers?

Reply:
Great question, Michelle and I ran into an issue where I undid the migration without undoing the seeds first so it got lost in limbo and GraphQL couldn't recognize that there was a new seed file to run so we had to change the number to 2.